spec/requests/racers_spec.rb

```diff
@@ -0,0 +1,111 @@
+require 'rails_helper'
+
+NUMBER_OF_RACERS = 3
+
+RSpec.describe 'Racers', type: :request do
+  describe 'GET /index' do
+    it 'returns http success' do
+      get '/racers/', headers: { 'Accept': 'application/json' }
+      expect(response).to have_http_status(:success)
+    end
+
+    context 'when there are no records' do
+      it 'returns an empty array' do
+        get '/racers/', headers: { 'Accept': 'application/json' }
+        expect(response.body).to eq('[]')
+      end
+    end
+
+    context 'when there are saved records' do
```

Comment:
you can have 2 contexts. One for when you have 1 racer (please don't use `record` since you're testing and documenting the `racer`) and the other when you have more than one racer.